cli/src/operation_templater.rs

```diff
@@ -249,6 +249,14 @@ fn builtin_operation_methods() -> OperationTemplateBuildMethodFnMap<Operation> {
         });
         Ok(L::wrap_string(out_property))
     });
+    map.insert(
+        "snapshot",
```

Comment:
I thought about `is_snapshot` for a sec, but everywhere else we have plain nouns for boolean getters.

Maybe `wc_snapshot`? The entire `working_copy_snapshot` does indeed seem super-long

Reply:
`wc_snapshot` works for me, and I agree that `snapshot` is also okay. My only concern was the method version `self.snapshot()` looks like a verb that returns some sort of snapshot of the `self`.

Since there isn't better alternative, maybe it's good to go with `snapshot`?

Reply:
Yeah I think snapshot is ok, keeping it